common/protob/messages-stellar.proto

```diff
@@ -123,6 +124,21 @@ message StellarManageOfferOp {
     optional uint64 offer_id = 7;                   // Offer ID for updating an existing offer
 }
 
+/**
+ * Request: ask device to confirm this operation type
+ * @next StellarTxOpRequest
+ * @next StellarSignedTx
+ */
+message StellarManageBuyOfferOp {
+    optional string source_account = 1;             // (optional) source account address
+    optional StellarAssetType selling_asset = 2;
+    optional StellarAssetType buying_asset = 3;
+    optional sint64 buy_amount = 4;
```

Comment:
please rename to `amount`

Reply:
You already asked for it the last time then closed it after I replied:

> The reason I used buyAmount is because it's what Stellar use (I guess it should be buy_amount by the way?). I know it's not consistent.
> 
> Do you want to switch to amount anyway?

I think it's better to stay consistent with the underlying implementation regardless of our opinion about it. (I asked them to change it into `amount` at a lower level)

Reply:
Right, sorry about that. Yes, please switch to `amount` anyway.